src/Quartz.Serialization.SystemTextJson/Calendars/DailyCalendarSerializer.cs

```diff
@@ -19,9 +19,9 @@ protected override DailyCalendar Create(JsonElement jsonElement, JsonSerializerO
 
     protected override void SerializeFields(Utf8JsonWriter writer, DailyCalendar calendar, JsonSerializerOptions options)
     {
+        writer.WriteBoolean(options.GetPropertyName("InvertTimeRange"), calendar.InvertTimeRange);
```

Comment:
This change was done to generate exactly the same JSON between STJ & Newtonsoft. This makes it possible to test generated JSON with Verify, to assert the same output is generated.

Might be something that is not possible in the long run, but for now only small changes were needed.